styles/global.css

```diff
@@ -125,8 +125,16 @@ em-emoji-picker {
   }
 }
 
-.line-compact {
-  line-height: calc(4 / 3 * 1em);
```

Comment:
Removed the unit `em` following the recommendation by MDN (https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#values):

> In most cases, **this is the preferred way** to set line-height and avoid unexpected results due to inheritance.